plaidbench/plaidbench/frontend_keras.py

```diff
@@ -198,9 +198,9 @@ def setup(self):
             eval(code, mod)
         self.x = mod['scale_dataset'](self.x)
         self.model = mod['build_model'](**build_model_kwargs)
-        if self.enable_bn_folding:
-            click.echo('Model loaded, folding in batch_norm')
-            self.model = self.fold_batch_norm(self.model)
+        # if self.enable_bn_folding:
+        #     click.echo('Model loaded, folding in batch_norm')
+        #     self.model = self.fold_batch_norm(self.model)
```

Comment:
Not certain, but perhaps it would be cleaner to disable this by changing line 341 to `enable_bn_folding = False` instead?

Reply:
Actually I'd like to get @jbruestle feedback on this to decide if we should just remove it entirely from plaidbench or what the plan is for this. I think we should try to write an MLIR pass for it. So I'm thinking this comment would just be temporary until we remove it.

Reply:
Do you want to wait for @jbruestle's feedback here for this PR, or shall we land this and have a followup PR if needed?

Reply:
I guess it wouldn't hurt to land it, but I still have build/ci issues to fix it seems.

Reply:
Honestly I'd leave the comment in for now, because in part I'm curious to try to track down while folding induced failure, and we may want to re-enable this if we fix the bug and haven't written a BN folding MLIR pass yet (which I agree is the long term solution).